aws/cloudformation/components/alarms.yml.erb

```diff
@@ -38,3 +38,23 @@
       Statistic: Maximum
       Threshold: 87
       TreatMissingData: missing
+<%  if environment == :production -%>
+  # Detect problems with the bin/cron/deliver_poste_messages scheduled job by monitoring outbound emails sent by AWS SES.
+  OutboundEmailAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      AlarmActions:
+        - "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-Urgent"
+      AlarmDescription: Send page if number of outbound transactional emails sent drops below threshold.
+      AlarmName: <%="#{stack_name}_no_outbound_emails_sent" %>
+      ComparisonOperator: Lower
+            MetricName: Send
+      Namespace: 'AWS/SES'
+      # No outbound transactional emails (password reset, new account welcome, etc.) sent per hour for 4 consecutive hours.
```

Comment:
Should we make this just a single Evaluation Period of 4 hours?

Reply:
Looking at three months of history on this metric, it looks like we don't drop below 25/hr unless there is a problem.  I would keep the 3600s period, and maybe even cut this down to 3 evaluation periods.